nav/load_waypoints/scripts/static_waypoints.json

```diff
@@ -1,44 +1,14 @@
-{ 
+{
```

Comment:
Why'd you delete most of this file ? We need to have all 4 waypoints of no man's land